proto/cel/expr/conformance/env_config.proto

```diff
@@ -0,0 +1,159 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package cel.expr.conformance;
+
+import "cel/expr/checked.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/descriptor.proto";
+
+option cc_enable_arenas = true;
+option go_package = "cel.dev/expr/conformance";
+option java_multiple_files = true;
+option java_outer_classname = "EnvironmentProto";
+option java_package = "cel.dev.expr.conformance";
+
+// Representation of a CEL Environment, defining what features and extensions
+// are available for conformance testing.
+message Environment {
+  // Name of the environment
+  string name = 1;
+
+  // Description for the current environment
+  string description = 2;
+
+  // Sets the namespace (container) for the expression.
+  // This is used to simplify resolution: e.g. with container
+  // `google.rpc.context`, an identifier of
+  // `google.rpc.context.AttributeContext` could be referred to simply as
+  // `AttributeContext` in the CEL expression.
+  string container = 3;
+
+  // Import represents a type name that will be abbreviated by its simple name
+  // using the cel.Abbrevs() option.
+  message Import {
+    // Qualified type name which will be abbreviated
+    string name = 1;
+  }
+
+  // List of abbreviations to be added to the CEL environment
+  repeated Import imports = 4;
+
+  // Set of options to subset a subsettable library
+  LibrarySubset stdlib = 5;
+
+  // List of extensions to enable in the CEL environment.
+  repeated Extension extensions = 6;
+
+  // ContextVariable represents a message type to be made available as a
+  // context variable to the CEL environment.
+  message ContextVariable {
+    // Fully qualified type name of the context proto.
+    string type_name = 1;
+  }
+
+  // If set, adds a context declaration from a proto message.
+  //
+  // Context messages have all of their top-level fields available as variables
+  // in the type checker.
+  repeated ContextVariable context_variables = 7;
```

Comment:
I know that previous representations used a repeated value here, but in practice implementations only supported a single context variable, so we should limit this to a single value. In test inputs, we only allow a single `context_proto` binding as well, so the cardinality would match.